bin/resty

```diff
@@ -47,6 +66,8 @@ GetOptions("c=i",             \(my $conns_num),
            "V|v",             \(my $version))
    or usage(1);
 
+check_arg_format('--shdict', qr/^[a-z]*\s\d+(?i)[km]$/, 'NAME SIZE', @shdicts);
```

Comment:
@thibaultcha Try larger values? I remember there is a minimum size restriction like 8KB or 12KB.